### ISSUE TITLE
Remove hardcoded values from Puppet manifests

### ISSUE THREAD

**Issue by hackdna** [refinery-platform/refinery-platform#1164]
Files: init.pp and aws.pp
Values: /vagrant/refinery, /var/lib, /opt, /data/import, /srv/refinery-platform/deployment/aws-requirements.txt, etc
Related to #1153 and potentially others

spurious error in /var/log/cloud-init-output.log:
```
17 gems installed
/var/lib/cloud/instance/scripts/part-001: 25: cd: can't cd to /vagrant/deployment
- GIT_BRANCH=acda478d8d2670c521b95cbd635bb483e08111bd
```

**Comment by mccalluc** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;0nakOtGR&#x2F;51-1164"><img src="https:&#x2F;&#x2F;github.trello.services&#x2F;images&#x2F;trello-icon.png" width="12" height="12"> 1164</a>

**Comment by hackdna** [comment]
Closed in #3208
